src/System.Security.Cryptography.Cng/ref/Configurations.props

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <PackageConfigurations>
       netcoreapp;
+      netstandard2.1;
```

Comment:
@ericstj with `netstandard2.1` ref can we stop building a ref assembly for `netcoreapp3.0`? For this project it seems the netcoreapp ref doesn't ship as part of the framework.

Reply:
That depends on what else is in the package.  Remember that netcoreapp-anything will have precedence over netstandard.  So if the package has *no* netcoreapp specific refs then you can get away with only netstandard.

Reply:
Just to confirm, if someone is building an asset with the `netcoreapp3.0` TFM and they reference this package, would they get `netcoreapp2.1` or `netstandard2.1` as the best match?

Reply:
Yeah, looks like the `netcoreapp2.0` is a better match for `netcoreapp3.0` than `netstandard2.1`, so we need to keep both tracks because of NuGet resolution rules.  (the current package has net46, net461, netcoreapp2.0, netstandard1.3, netstandard2.0 as the refs)

Reply:
Ok, thanks for checking. Then let's leave as is.

Reply:
I don't suppose there is anyway you can simplify the package to only use netstandard versions?  Would this drop api from netcoreapp2.0?

Reply:
> I don't suppose there is anyway you can simplify the package to only use netstandard versions? Would this drop api from netcoreapp2.0?

Currently the netcoreapp2.1 line has three additional types and two additional members.  The biggest problem would be if the implementation fell back to netstandard2.0 then it would drop support for DSA across an upgrade.  So we can't really drop that line, either.

If only netstandard2.0 had been aligned with net462 (or, better, net472) :)